event-catalog/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Build Stage
+FROM node:16-alpine AS BUILD_IMAGE
```

Comment:
Can this use latest version of node? Node 20.

Reply:
updated